### PR TITLE
Bump SBOM version so the file output is sbom-desktop-vX.Y.Z.json

### PR DESCRIPTION
**Pull request by enzowritescode**
#### Summary

Change SBOM file from `sbom.json` to `sbom-desktop-vX.Y.Z.json`

#### Ticket Link

https://mattermost.atlassian.net/browse/SEC-7235

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/snyk-sbom.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@ jobs:
   sbom:
     permissions:
       contents: write
-    uses: mattermost/actions-workflows/.github/workflows/snyk-sbom.yml@26d71413024508c19579a7df94d93488c4fe3bbf
+    uses: mattermost/actions-workflows/.github/workflows/snyk-sbom.yml@e0a2a4d22c19d9596772fa1472ee88c94f2de6ea
     secrets: inherit
     with:
       is_monorepo: false
+      repo_name: desktop
```
Comment (mvitale1989) [inline .github/workflows/snyk-sbom.yml:14]: This line may be removed after this is merged: https://github.com/mattermost/actions-workflows/pull/52

Comment (enzowritescode) [reply .github/workflows/snyk-sbom.yml:14]: Yeah I will need to remove that and then update the commit hash. I have it staged locally.

Comment (enzowritescode) [reply .github/workflows/snyk-sbom.yml]: @mvitale1989 done

**Review by mvitale1989 — APPROVED** [review]

**Pull request merged**